metrics/prometheus.go

```diff
@@ -1544,6 +1544,19 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 
 	}
 
+	if includedMetrics.Has(container.WssMetric) {
+		c.containerMetrics = append(c.containerMetrics, []containerMetric{
+			{
+				name:      "container_wss",
+				help:      "Container referenced bytes during last measurements cycle",
+				valueType: prometheus.GaugeValue,
+				getValues: func(s *info.ContainerStats) metricValues {
+					return metricValues{{value: float64(s.Wss), timestamp: time.Now()}}
+				},
+			},
+		}...)
```

Comment:
This is only one metric. Can't it be appended without using a slice?

Reply:
It's generic mechanism for creating prometheus metrics per container. I think that here consistency should be kept.